handler.py

```diff
@@ -105,72 +159,79 @@ def feedback_from_cases(response, params, cases):
     eval_function_feedback = []
     for i, case in enumerate(cases):
         # Validate the case block has an answer and feedback
-        if 'answer' not in case:
+        if "answer" not in case:
             warnings += [{"case": i, "message": "Missing answer field"}]
             continue
 
-        if 'feedback' not in case:
+        if "feedback" not in case:
```

Comment:
These tokens (e.g.`"answer"`, `"feedback"` etc.) seem to be crucial to the warning-collation process. How about extracting them into an enum?